Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/zipline .
+# 
+# To run the container:
+#
+#    docker run -v=/path/to/your/notebooks:/projects -p 8888:8888/tcp --name zipline -it quantopian/zipline
```

Comment:
This seems to be an interaction between the server with unsigned certificate and your browser.  What OS/browser are you using?  When you connect to the notebook server do you connect to https://127.0.0.1:8888, and do you get a warning about unsafe site or similar?  

Since the server is probably only going to accessed used locally, we could probably revert to an HTTP server with no certificate.  I was tinkering with SSL/TLS for another project and thought everything was OK, so I dropped it in this one.  It’s probably not required here.  Let me know if you agree/disagree.